src/parents.js

```diff
@@ -43,5 +80,15 @@ export function set (node, parent, { force } = {}) {
  * @returns {object | undefined} The parent node, or undefined if the node has no parent
```

Comment:
Wouldn't we get `null` in that case? We get `undefined` when we don't *know* the parent

Reply:
I'll clear this up